### ISSUE TITLE
[Data]: ALS Panama Canal Zone 2023

### ISSUE THREAD

**Issue by samuelgrubinger** [MAAP-Project/ZenHub#1127]
### Dataset Name

ALS Panama Canal Zone 2023

### Dataset Description

2023 high-resolution airborne LiDAR data for Barro Colorado Island and other Smithsonian ForestGEO Sites in Central Panama

### Requestor Name/Affiliation

Samuel Grubinger, University of Maryland

### Platform/Method/Sensor

LiDAR data were collected with an IGI LiteMapper 6800i, which incorporates a Riegl LMS680 full waveform sensor

### URL or DOI to Dataset Description

https://smithsonian.dataone.org/view/doi:10.60635/C34W2W

### URL to Download or Access the Data

https://smithsonian.dataone.org/datasets/ALS_Panama_2023/

### Data License

Yes

### Intended Science Use Case

These ALS data will be used alongside plot census data from several sites in the Canal Zone in central Panama to generate aboveground biomass maps as part of the GEO-TREES project. These sites will be some of the first of many, used for demonstration and methods development. 

### Additional Information

David Minor may already have downloaded some of these data, or may have submitted a data request already. He is on paternal leave, so I have not been able to talk with him.

**Comment by wildintellect** [comment]
@samuelgrubinger thanks for the submission. David did not ask for our help on this previously. I think it makes sense for us to ingest this data to the MAAP STAC, partly so we can convert to Cloud Optimized Point Cloud (COPC), and also fix the missing CRS issue I observed on a few of the files.

```
$ pdal info BCI_2023_623500_1011500.laz 
PDAL: readers.las: Global encoding WKT flag not set for point format 6 - 10.
```
https://gis.stackexchange.com/questions/413191/python-pdal-error-reading-format-1-4-las-file-readers-las-error-global-enco

@omshinde can take lead on making this happen.

**Comment by samuelgrubinger** [comment]
Hi Alex,
That would be extremely helpful! Thanks so much.
Sam

**Comment by omshinde** [comment]
We pushed the data files to the MAAP Test STAC - https://stac-browser.dit.maap-project.org/collections/ALS_GROUND_COPC. If these are the files you're looking for then we can push these to the dev STAC (production).

**Comment by omshinde** [comment]
@samuelgrubinger I've prepared a notebook for you to access the ALS COPC files we have on MAAP STAC (DIT). Can you please access and let us know if this is the data you're looking for? If these files are separate from what you need, then we can start ingesting new data files. 

Link: https://ade.maap-project.org/serverz9ssxsc3-ws-jupyter/server-3100/lab/tree/my-public-bucket/als_data_access_STAC.ipynb

I'll wait for your confirmation before ingesting new LiDAR data files.

**Comment by omshinde** [comment]
Based on discussion with @samuelgrubinger 

- The current ALS files in the MAAP Test STAC - https://stac-browser.dit.maap-project.org/collections/ALS_GROUND_COPC are ground classified version. Also, a couple of weird instances of `high NumberOfReturns` and `ReturnNumber > NumberOfReturns` are present.
- Thus, it will be good to proceed with ingesting data from https://smithsonian.dataone.org/datasets/ALS_Panama_2023/
- Note: Ensure proper citation is added in the STAC records (Ref: https://smithsonian.dataone.org/view/doi%3A10.60635%2FC34W2W)

**Comment by omshinde** [comment]
@samuelgrubinger I have a question-

- From [here](https://smithsonian.dataone.org/datasets/ALS_Panama_2023/), do we want to ingest all the directories to the STAC or only the files for the BCI?

**Comment by samuelgrubinger** [comment]
Yes, all the ALS acquisitions, please. I don't need all the outputs, just the LAZ files. Thanks!
